Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -209,7 +210,8 @@ public function build(array $buildSubject): array
             unset($requestBody['installments']);
         }
 
-        $requestBody['additionalData']['allow3DS2'] = true;
+        $requestBody['additionalData']['allow3DS2'] =
+            $this->configHelper->getThreeDSMode($storeId) === ThreeDSMode::THREEDS_MODE_NATIVE;
```

Comment:
Won't this will be false if the new config value is not yet set?
I think we should keep the default behaviour until the merchant sets the config to redirect

Reply:
The default value will always be `native` as we have set it in `config.xml`.